go/ql/lib/semmle/go/frameworks/K8sIoApimachineryPkgRuntime.qll

```diff
@@ -297,83 +136,29 @@ module K8sIoApimachineryPkgRuntime {
     override DataFlow::FunctionOutput getOutput() { result.isParameter(0) }
 
     override string getFormat() { result = "protobuf" }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp = this.getAnInput() and outp = this.getOutput()
-    }
-  }
-
-  private class RawExtensionDeepCopy extends TaintTracking::FunctionModel, Method {
-    RawExtensionDeepCopy() { this.hasQualifiedName(packagePath(), "RawExtension", "DeepCopy") }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp.isReceiver() and outp.isResult()
-    }
-  }
-
-  private class RawExtensionDeepCopyInto extends TaintTracking::FunctionModel, Method {
-    RawExtensionDeepCopyInto() {
-      this.hasQualifiedName(packagePath(), "RawExtension", "DeepCopyInto")
-    }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp.isReceiver() and outp.isParameter(0)
-    }
   }
 
-  private class RawExtensionMarshal extends TaintTracking::FunctionModel, Method,
-    MarshalingFunction::Range
-  {
+  private class RawExtensionMarshal extends Method, MarshalingFunction::Range {
     RawExtensionMarshal() { this.hasQualifiedName(packagePath(), "RawExtension", "Marshal") }
 
     override DataFlow::FunctionInput getAnInput() { result.isReceiver() }
 
     override DataFlow::FunctionOutput getOutput() { result.isResult(0) }
 
     override string getFormat() { result = "protobuf" }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp = this.getAnInput() and outp = this.getOutput()
-    }
   }
 
-  private class RawExtensionUnmarshal extends TaintTracking::FunctionModel, Method,
-    UnmarshalingFunction::Range
-  {
+  private class RawExtensionUnmarshal extends Method, UnmarshalingFunction::Range {
     RawExtensionUnmarshal() { this.hasQualifiedName(packagePath(), "RawExtension", "Unmarshal") }
 
     override DataFlow::FunctionInput getAnInput() { result.isReceiver() }
 
     override DataFlow::FunctionOutput getOutput() { result.isParameter(0) }
 
     override string getFormat() { result = "protobuf" }
-
-    override predicate hasTaintFlow(DataFlow::FunctionInput inp, DataFlow::FunctionOutput outp) {
-      inp = this.getAnInput() and outp = this.getOutput()
-    }
-  }
-
-  private class UnknownDeepCopy extends TaintTracking::FunctionModel, Method {
-    UnknownDeepCopy() {
-      this.hasQualifiedName(packagePath(), "Unknown", ["DeepCopy", "DeepCopyObject"])
```

Comment:
I don't see a model for `Unknown.DeepCopyObject`.

Reply:
This is inherited from `Object.DeepCopyObject`.